conan/tools/cmake/toolchain/blocks.py

```diff
@@ -502,6 +505,38 @@ class FindFiles(Block):
         {% endif %}
     """)
 
+    @staticmethod
+    def _multiconfig_generator(prop):
+        return ' '.join(f'"$<$<CONFIG:{c}>:{i}>"' for c, v in prop.items() for i in v)
+
+    def _get_host_runtime_dirs_mc(self, host_req):
+        settings = self._conanfile.settings
+        host_runtime_dirs = {}
+        is_win = self._conanfile.settings.get_safe("os") == "Windows"
+
+        # Get the previous configuration
+        if os.path.exists(CONAN_TOOLCHAIN_FILENAME):
+            existing_include = load(CONAN_TOOLCHAIN_FILENAME)
+            pattern_lib_dris = r"list\(PREPEND CONAN_RUNTIME_LIB_DIRS ([^)]*)\)"
+            msvc_runtime_value = re.search(pattern_lib_dris, existing_include)
```

Comment:
`msvc_runtime_value` the variable name does not really reflect what it holds, this applies to all platforms "list of directories that contain shared libraries needed at runtime"